src/Models/WebMapServiceCatalogGroup.js

```diff
@@ -323,6 +330,13 @@ function createWmsDataSource(wmsGroup, layer, supportsJsonGetFeatureInfo, suppor
         result.description += layer.Abstract;
     }
 
+    if (typeof(wmsGroup.itemProperties) === "object") {
```

Comment:
We use single quotes for strings throughout.  Also, the parenthesis after typeof are unnecessary.